insert_libzstd_ifdefs.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+import sys
+import glob
+
+FLAG="USE_EXTERNAL_ZSTD"
+
+def insert_ifdefs(file):
+
+    with open(file, "r") as fd:
+        content=fd.read()
+
+    with open(file, "w") as fd:
+        fd.seek(0)
```

Comment:
I'm *pretty* sure this is unnecessary, since opening a file for writing and not appending should start at the beginning of the file? Doesn't really matter though ...

Reply:
fixed